math/algebraic_test.mbt

```diff
@@ -56,5 +56,5 @@ test "minimum.ref" {
 
 // For testing purposes
 priv struct T {
-  mut x : Int
```

Comment:
The `mut` is indented to make compiler unable to do unbox optimization on type `T`.

Reply:
While there's no something likes `//@warning "-unused-mut"`. I think we need a manual assignment statement to eliminate the warning?

Reply:
Currently `T` will not be unboxed globally even if its only field is immutable. If some value of `T` is used locally by accessing field only then that particular value may be unboxed, but that's not the case here, and mutable structs also get unboxed in that case.

This test itself may be controversial but let's discuss and fix that in a separate PR